actionview/lib/action_view/helpers/rendering_helper.rb

```diff
@@ -1,5 +1,7 @@
 # frozen_string_literal: true
 
+# :markup: markdown
```

Comment:
I think if we want to switch the file to markdown we should go all the way, there are a lot of RDoc-isms left here (`=` header, `<tt>`, etc.)

![image](https://github.com/user-attachments/assets/03d88b8d-8c51-4ca8-8822-43f8e4b6943b)

Reply:
Thanks! I ran `tools/rdoc-to-md`

Reply:
@skipkayhil Do you think it's good to go now?